tests/rptest/services/compacted_verifier.py

```diff
@@ -0,0 +1,222 @@
+# Copyright 2020 Redpanda Data, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.services.service import Service
+import requests
+from time import sleep
+import sys
+import json
+from rptest.util import wait_until
+from enum import Enum
+
+
+class CrushedException(Exception):
+    pass
+
+
+class ConsistencyViolationException(Exception):
+    pass
+
+
+OUTPUT_LOG = "/opt/remote/var/rw.log"
+
+
+class Workload(Enum):
+    IDEMPOTENCY = 1
+
+
+workloads = {Workload.IDEMPOTENCY: "io.vectorized.compaction.idempotency.App"}
+
+
+class CompactedVerifier(Service):
+    logs = {"rw_stdout_stderr": {"path": OUTPUT_LOG, "collect_default": True}}
+
+    def __init__(self, context, redpanda, workload):
+        super(CompactedVerifier, self).__init__(context, num_nodes=1)
+        self._redpanda = redpanda
+        self._is_done = False
+        self._node = None
+        if workload not in Workload:
+            raise Exception(f"Unknown workload {workload}")
+        self._workload = workload
+
+    def is_alive(self, node):
+        result = node.account.ssh_output(
+            "bash /opt/remote/control/alive.sh rw")
+        result = result.decode("utf-8")
+        return "YES" in result
```

Comment:
I think this isn't really solving our issue related with `SSHTimeouts`, as this depends in an equal way on SSH as the previous compacted-topic-verifier. The verifier that was removed in one of the previous commits doesn't stream the data over SSH but stores the state in a state file on a local node, then the state file was used to compare produced data with consumed one.

Reply:
Now i see that the `CompactionEndToEndTest` is the one that was causing tiemouts as it was based on Verifiable Consumer/Producer pair